pkg/workloads/console/runner/runner.go

```diff
@@ -211,6 +228,56 @@ func (c *Runner) WaitUntilReady(ctx context.Context, createdCsl workloadsv1alpha
 	}
 }
 
+func (c *Runner) waitForRoleBinding(ctx context.Context, csl *workloadsv1alpha1.Console) error {
```

Comment:
This is quite tightly coupled to our implementation of our controller. Do you think it might be better if we use a `SelfSubjectAccessReview` to instead ask the cluster if I have permissions to attach? That's asking a more direct question of whether we're ready and able to continue.

Reply:
I'll look into this, I didn't know about SelfSubjectAccessReview. I agree it's a more blackbox way of verifying behaviour!

Reply:
Similar to thread below, let's merge this now because it works, and I'll PR this improvement v soon.